cmd/external-repos/main.go

```diff
@@ -130,3 +131,35 @@ func scanForExternalRepos(path string) {
 	}
 	log.Info().Msg("Saved External Repositories")
 }
+
+func enqueueIntrospectAllRepos() error {
+	q, err := queue.NewPgQueue(db.GetUrl())
+	if err != nil {
+		return fmt.Errorf("error getting new task queue: %w", err)
+	}
+	c := client.NewTaskClient(&q)
+
+	repoDao := dao.GetRepositoryDao(db.DB)
+	repos, err := repoDao.List(true)
+	if err != nil {
+		return fmt.Errorf("error getting repositories: %w", err)
+	}
+	for _, repo := range repos {
+		t := queue.Task{
+			Typename: tasks.Introspect,
+			Payload: tasks.IntrospectPayload{
+				Url: repo.URL,
+			},
+			RepositoryUUID: repo.UUID,
+		}
+		_, err = c.Enqueue(t)
+		if err != nil {
+			log.Err(err).Msgf("error enqueueing introspecting for repository %v", repo.URL)
+		}
+	}
+	err = repoDao.OrphanCleanup()
+	if err != nil {
+		log.Err(err).Msg("error during orphan cleanup")
+	}
+	return nil
+}
```

Comment:
I kinda feel like this method should go in external_repos somewhere, what do you think?

Reply:
I thought I had a reason to put it here, but I now I can't remember. Maybe not. I'm gonna try moving it and see if I don't hit some kind of dependency error.